tests/unit/ui/components/TestExpandableCard.js

```diff
@@ -32,24 +44,17 @@ describe(__filename, () => {
   });
 
   it('toggles when clicked', () => {
-    // We have to mount for this test because of the usage of `setState`
-    // in this component.
-    const root = mount(<ExpandableCard i18n={fakeI18n()} />);
-    const card = () => root.find('.ExpandableCard');
-    const fakeEvent = createFakeEvent();
+    const { store } = dispatchClientMetadata();
 
-    // Clicking on the toggle should expand the card.
-    root.find('.ExpandableCard-ToggleLink').simulate('click', fakeEvent);
+    const root = render({ store });
 
-    sinon.assert.called(fakeEvent.preventDefault);
-    expect(card()).toHaveClassName('ExpandableCard--expanded');
+    // We want to simulate card's link onclick.
+    // TODO: look into why this isn't available with find
+    root.instance().onClick(createFakeEvent());
```

Comment:
I know maybe this isn't the way I should be doing this but when I do something like this:

`root.find(Card).simulate('click', createFakeEvent());` or looked for the `a` tag or the a's class (root.find('.ExpandableCard-ToggleLink')..., it didn't work :/

Should I be looking into mount here?